### ISSUE TITLE
Installed successfully but rendering is failed

### ISSUE THREAD

**Issue by EasonPai** [adobe/aem-guides-wknd#88]
I have successfully installed all modules onto my instance (6.5.3).

![image](https://user-images.githubusercontent.com/678352/74597401-f3e04100-5099-11ea-8d0f-b364fba03209.png)

But pages cannot render normally as the reference site do.

![image](https://user-images.githubusercontent.com/678352/74597443-a0222780-509a-11ea-974a-137af59272ed.png)

I can see there are some warnings when rendering this page:
```
16.02.2020 09:00:20.279 *WARN* [192.168.1.1 [1581814818862] GET /content/wknd/language-masters/en.html HTTP/1.1] org.apache.sling.caconfig.resource.impl.def.DefaultConfigurationResourceResolvingStrategy Ignoring reference to /conf/wknd/settings from /conf/wknd/settings - Probably misconfigured as it ends with '/settings'
16.02.2020 09:00:20.288 *WARN* [192.168.1.1 [1581814818862] GET /content/wknd/language-masters/en.html HTTP/1.1] org.apache.sling.caconfig.resource.impl.def.DefaultConfigurationResourceResolvingStrategy Ignoring reference to /conf/wknd/settings from /conf/wknd/settings - Probably misconfigured as it ends with '/settings'
16.02.2020 09:00:20.305 *WARN* [192.168.1.1 [1581814818862] GET /content/wknd/language-masters/en.html HTTP/1.1] org.apache.sling.caconfig.resource.impl.def.DefaultConfigurationResourceResolvingStrategy Ignoring reference to /conf/wknd/settings from /conf/wknd/settings - Probably misconfigured as it ends with '/settings'
```

How can I fix the page rendering? Need some help here, thanks!

Regards,
Eason

**Comment by godanny86** [comment]
hi @EasonPai it looks like the template policies (which includes the styles) are not being applied correctly... what branch did you deploy?

You may want to delete the content in your local AEM instance at `/content/wknd` and `/conf/wknd` and try to re-deploy the project:

```shell
mvn -PautoInstallSinglePackage clean install
```

**Comment by EasonPai** [comment]
Hi @godanny86 

Thanks for the help.  I am using Master branch. I tried couple times to delete  and install /content/wknd & /conf/wknd back again. But the result and the warnings are the same. 

Later on i just randomly install every single modules couple times, and eventually I got the same result as what it supposed to be and no warnings!!! I have no idea what is happening behind the scene, but surely i don't wanna remove and try them again.

It helps anyway, thanks!

regards,
Eason
